joomla/joomla-cms
# [4.0] ES6 Please com_contenthistory

Pull Request for Issue # .

### Summary of Changes
Refactor script to ES6


### Testing Instructions
Check that versioning still functions as before
![screenshot 2018-08-07 at 21 13 21](https://user-images.githubusercontent.com/3889375/43797288-31172400-9a87-11e8-991a-1d8861bc6305.png)


PS there is a bug with the modals in the toolbar so you might have to delete the last div (class backdrop)
### Expected result



### Actual result



### Documentation Changes Required
